vapi/library/library.go

```diff
@@ -146,6 +200,18 @@ func (c *Manager) SyncLibrary(ctx context.Context, library *Library) error {
 	return c.Do(ctx, url.Request(http.MethodPost), nil)
 }
 
+// PublishLibrary publishes the library to specified subscriptions.
+// If no subscriptions are specified, then publishes the library to all its subscriptions.
+func (c *Manager) PublishLibrary(ctx context.Context, library *Library, subscriptions []string) error {
```

Comment:
This isn't part of my use case, but wanted to point out that it's also possible (through the vcenter console) to publish a specific library item to a specific subscription.  Might be useful for others to have this.  Having the blanket publish all library items to all subscriptions is the functionality that I'm after.

Reply:
I had seen that sync supports per-item (`library.sync` makes use of that), but didn't see that for publish. I'll take another look before merging this.  Thanks for the review!

Reply:
Found the per-item publish in the UI+API and updated the branch with support for that. Can see how that'd be useful.